oap-ape/ape-native/src/com_intel_ape_ParquetReaderJNI.cc

```diff
@@ -103,3 +106,9 @@ JNIEXPORT jboolean JNICALL
 Java_com_intel_ape_ParquetReaderJNI_isNativeEnabled(JNIEnv* env, jclass cls) {
   return ape::Reader::isNativeEnabled();
 }
+
+JNIEXPORT jlong JNICALL Java_com_intel_ape_ParquetReaderJNI_createPlasmaClientPool(
+    JNIEnv* env, jclass cls, jint capacity) {
+  ape::PlasmaClientPool* pool = new ape::PlasmaClientPool(capacity);
```

Comment:
if possible, add a graceful close/free method for PlasmaClientPool

Reply:
good suggestion. will add it